packages/table-react/documentation/ExpandableTableExample.tsx

```diff
@@ -14,7 +14,7 @@ const rows = [
 const ExpandableTableExample: VFC<ExampleComponentProps> = ({ boolValues, choiceValues }) => {
     const compact = boolValues?.["Kompakt"];
     const headless = boolValues?.["Skjul overskrift"];
-    const clickable = boolValues?.["Klikkbar"];
```

Comment:
Både Klikkbar (uten markering ved klikk) og Markér ved klikk er mulige varianter, så det hadde vært fint med både og her i eksempelet

Reply:
Beggars can't be choosers 🤣